omarCA/wraith
# Haunt

Currently, running wraith haunt will: 
1.initialize the static folder into a repo and add gitUrl as origin
2.commits folder, pushes to origin master

To test:
1. Create blank repo on github 
2. Insert url into gitUrl (line 7)
3. run wraith possess
4. run wraith haunt

issues:
Should not create static folder within a previous git repo path. The static folder should be created in a non git folder path to avoid git push conflicts.
